.travis.yml

```diff
@@ -45,12 +27,11 @@ before_script:
   - if [ "$SKIP_INTEROP" != "" ]; then composer remove --dev --no-update container-interop/container-interop; fi;
   - if [ "$DEPENDENCIES" != "low" ]; then composer update; fi;
   - if [ "$DEPENDENCIES" = "low" ]; then composer update --prefer-lowest; fi;
-  - if [ ! "$PHP_VERSION" ]; then export PHP_VERSION=`php -r 'echo PHP_MAJOR_VERSION;'`; fi;
   - export PATH=./bin:$PATH
 
 script:
   - ./vendor/bin/phpunit
-  - behat -fprogress --strict --tags "${SKIP_INTEROP}~@php-version,@php${PHP_VERSION}"
+  - behat -fprogress --strict --tags "${SKIP_INTEROP}~@random"
```

Comment:
We only need to pass `--tags "~@interop"` for build skipping the interop, any ideas how to make it cleaner?

Reply:
Remove the `&&` from the var and then something like `$([ -z "$SKIP_INTEROP" ] || echo "$SKIP_INTEROP&&")` (note outer parentheses)